aten/src/ATen/native/cudnn/MHA.cpp

```diff
@@ -754,6 +756,16 @@ void run_cudnn_SDP_bprop(
       "cuDNN SDPA expected grad_output.strides() == output.strides(), "
       "the previous step probably failed to materialize a grad_output "
       "with matching strides...");
+#else
+  const auto innermost_dO_stride = dO.strides()[dO.strides().size() - 1];
```

Comment:
Looks good, we have a test covering this right?

Reply:
This is covered yes, in a number of places because `.sum().backward()` is used in `test_transformers.py`